airflow/container.go

```diff
@@ -131,10 +134,10 @@ func generateConfig(projectName, airflowHome, envFile string, imageLabels map[st
 		return "", err
 	}
 
-	var triggererEnabled bool
-	airflowVersion := imageLabels[airflowVersionLabelName]
-	if versions.GreaterThanOrEqualTo(airflowVersion, triggererAllowedAirflowVersion) {
-		triggererEnabled = true
+	// check if Airflow triggerer is enabled, on AC images, look if version is > 2.1.0, for runtime if version >= 4.1.0
+	triggererEnabled, err := CheckTriggererEnabled(airflowHome, dockerfile, runtimeVersionCheck)
```

Comment:
any reason to move away from image labels to parsing to the dockerfile? I feel picking version from image label is more reliable compared to parsing dockerfile

Reply:
Why is it more reliable ? The Airflow version is added to 2 places:
- On the actual tag of the image, which can be found by simply reading from a file, which is always present inside an astronomer/airflow project (Dockerfile)
- The label on the image, which needs to be first pulled, then docker inspect.

Also, I made the decision of moving to reading Dockerfile since runtime code in Cloud-cli already uses this approach, and I wanted to use a consistent approach for both AC and Runtimes.

Reply:
> Why is it more reliable? 

Because users can use custom images built on top of AC or Runtime image, in which case it is not necessary for users will carry forward the same version tag, but those images will inherit label from the base image.

> The label on the image, which needs to be first pulled, then docker inspect

We don't need to separately pull the image in this case, since we are already building the image from Dockerfile beforehand.

Reply:
OK I see, in that case I'll use labels since customer can use a custom image.

Reply:
@AmFlint  / @neel-astro may be it can help https://github.com/astronomer/astro-runtime/blob/main/image/Dockerfile#L31

Reply:
I'll take a look at this to see if it's reliable for runtimes too

Reply:
Added for both AC and Runtimes